functions/api/transactions.js

```diff
@@ -181,4 +182,6 @@ async function hasNextPage (transactionCollection, snapshot){
     });
 }
 
-module.exports = router;
\ No newline at end of file
+module.exports = router;
+module.exports.hasPreviousPage = hasPreviousPage;
```

Comment:
suggestion: its common to both transaction and audit. The funcs should be in a common place and not in one of the files.

Reply:
You can move it to utils.js. I have also added getNextDate and getPrevDate functions there.